Scripts/Python/xHighLevelStarTrekDoor.py

```diff
@@ -68,19 +78,14 @@ def __init__(self):
         self.version = 3
 
         PtDebugPrint("DEBUG: xHighLevelStarTrekDoor.__init__: v. %d" % self.version)
-        
-        self.PrevDoorState = 0
-        self.DoorState = 0
-        self.DoorStack = []
+
         self.DoorEnabled = 1
-        self.respondertime = 30
+        rgnSensor.volumeSensorNoArbitration(True)
```

Comment:
`rgnSensor` does not actually have the key of the `plVolumeSensorConditionalObject` yet, so this has no effect. You'd want to put this in a `def OnInit(self):` method.